expo/gbe/models.py

```diff
@@ -344,6 +344,10 @@ def schedule(self):
             resources_allocated__resource__worker___item=self)]
         return sorted(set(events), key=lambda event: event.start_time)
 
+    def volunteer_schedule(self, conference=None):
+        conference = conference or Conference.current_conf()
```

Comment:
I worry about adding to the auto-selection of conference at the model level, because when I started looking at the django-debug-toolbar -we had conference selection happening all over the place.
I'm not ready to call it a blocker, because I haven't dug in far enough on the analysis of where and how it's happening... but I'd like us to figure out at what level we detect the target conference, and then do it in one place consistently.